brainscore_language/data/pereira2018_v2022/__init__.py

```diff
@@ -0,0 +1,23 @@
+import logging
```

Comment:
Let's talk about whether this goes into its own file or is part of the existing pereira2018 data plugin (I put a note on the agenda)